pkg/tuned/host_payload.go

```diff
@@ -94,7 +93,8 @@ func ProvideIgnitionFiles(stalld bool) []ign3types.File {
 	files := []ign3types.File{}
 
 	if stalld {
-		files = append(files, stalldIgnitionFile())
+		files = append(files, ignitionFile("/usr/local/bin/stalld", 0755))
+		files = append(files, ignitionFile("/usr/local/bin/throttlectl.sh", 0755))
```

Comment:
what will happen in upgrade if the file needs to change/remove ?

Reply:
Those files are automatically wiped from the host on a profile change if/when `stalld` is not needed by a selected profile.  During an upgrade/profile change the operator will compare the old/new ignition files and update them accordingly.  I don't believe there is a problem here, is there?

Reply:
no, just double checking